ch390/idf_component.yml

```diff
@@ -0,0 +1,13 @@
+version: "0.1.0"
+targets:
+  - esp32
+  - esp32s2
+  - esp32c3
+  - esp32s3
+  - esp32c2
+  - esp32c6
+  - esp32h2
```

Comment:
If the component doesn't have target compatibility restrictions, it is better to not specify `targets:` at all — that way we don't have to remember to update this list.

Reply:
Certainly! Besides, I also strongly recommend to remove target limit in `LAN867x`. As a direct result, the example in the `ethernet_init` can only run under `ESP32` (even if I don't want to use `LAN867x`), otherwise the compilation will be failed.